box_it_script/box_it.js

```diff
@@ -0,0 +1,72 @@
+#!/home/nimbus-user/.nvm/versions/node/v17.6.0/bin/node
+function drawLine(num){
+    let final =''
+    for (let i = 0; i < num; i++){  
+        final = final + '━';
+    }
+    return final;
+}
+
+function drawTopBorder(num){
+    let len = drawLine(num)
+    let final ='┏' + len + '┓'
+    return final;
+}
+function drawMiddleBorder(num){
+    let len = drawLine(num)
+    let final ='┣' + len + '┫'
+    return final;
+}
+function drawBottomBorder(num){
+    let len = drawLine(num)
+    let final ='┗' + len + '┛'
+    return final;
+}
+
+function drawBarsAround(string){
+    let final ='┃' + string + '┃'
+    return final;
+}
+
+function boxIt(arr){
+    let final =''
+    let br ='\n'
+    let longest =0
+for (let i = 0; i < arr.length; i++) {
+    if(arr[i].length > longest){
+        longest = arr[i].length; 
+    }
+}
+
+for (let i = 0; i < arr.length; i++) {
+    let space = ' '
```

Comment:
Preferred to use `const` if you are not going to change the value of the variable.